cmd/nodeup/main.go

```diff
@@ -64,17 +69,49 @@ func main() {
 	retries := flagRetries
 
 	for {
-		cmd := &nodeup.NodeUpCommand{
-			ConfigLocation: flagConf,
-			Target:         target,
-			CacheDir:       flagCacheDir,
-			FSRoot:         flagRootFS,
-			ModelDir:       models.NewAssetPath("nodeup"),
-		}
-		err := cmd.Run(os.Stdout)
-		if err == nil {
-			fmt.Printf("success")
-			os.Exit(0)
+		var err error
+		if install {
+			// create a systemd unit to bootstrap kops
+			// using the same args as we were called with
+			var command []string
+			for i := 0; i < len(os.Args); i++ {
+				s := os.Args[i]
+				if s == "-install" || s == "--install" {
+					continue
+				}
+				if i == 0 {
+					// We could also try to evaluate based on cwd
+					s, err = os.Readlink("/proc/self/exe")
+					if err != nil {
+						glog.Fatalf("error reading /proc/self/exe link: %v", err)
+					}
+				}
+				command = append(command, s)
+			}
+			i := bootstrap.Installation{
+				MaxTaskDuration: 5 * time.Minute,
+				CacheDir:        flagCacheDir,
+				Command:         command,
+				FSRoot:          flagRootFS,
+			}
+			err = i.Run()
```

Comment:
Doesn't look like we do anything if we get an Error back from `i.Run()` can we at least log it please?